components/web/php/php-8_1-ext-memcache/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Sergey Avseyev
+# Copyright (c) 2019 Alexander Pyhalov
+# Copyright (c) 2022 Friedrich Kink
+#
+
+BUILD_BITS=             64
+USE_OPENSSL11=  yes
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		memcache
+COMPONENT_VERSION=	8.0
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	https://pecl.php.net/package/memcache
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tgz
+COMPONENT_ARCHIVE_HASH=	sha256:defe33e6f7831d82b7283b95e14a531070531acbf21278f3f0d7050505cf3395
+COMPONENT_ARCHIVE_URL=	https://pecl.php.net/get/$(COMPONENT_ARCHIVE)
+COMPONENT_SUMMARY=	php memcache extension
+COMPONENT_FMRI=		web/php-81/extension/php-memcache
+COMPONENT_LICENSE=	PHP3.01
+COMPONENT_LICENSE_FILE=	LICENSE
+COMPONENT_CLASSIFICATION=	Development/PHP
+
+include $(WS_MAKE_RULES)/common.mk
```

Comment:
gmake test doesn't seem to work. Please eithter fix that or add
test_target= $(NO_TESTS)
here.